### PR TITLE
Update README to also describe the Py 3.7 and Py 3.8 images

### LINKED ISSUE #33: readme: update documentation to reflect python 3.7 and 3.8 images

### PR DESCRIPTION
**Pull request by alexdutton**
I don't know whether the non-3.6 images have automated builds, so I've leftthat section alone, making sure it only refers to Py 3.6.

Resolves #33.

### CODE REVIEW

**Review by ppanero — APPROVED** [review]
LGTM! Thanks for this. Could you add the rolling builds of 3.7 and 3.8? the rules are exactly the same, but with the .7 and .8 instead of .6 :)

🚀 Might also need rebase, since I merged your other PR

**Comment by alexdutton** [comment]
It currently says that `python3.6/Dockerfile` gets built and tagged with the git tag name. Should this be `3.6-{tag}` from now on, with the same pattern for other versions?

I'm going to hold off changing the README for now given the implication that all three Dockerfiles are tagged identically.

**Comment by ppanero** [comment]
@alexsdutton hmm I see your point. However, I don't think they were ever `{tag}`... I think that was an error. If you see
- here: https://hub.docker.com/r/inveniosoftware/centos7-python/tags
- and here: https://hub.docker.com/r/inveniosoftware/centos8-python/tags

Which aligns with the current CI: https://github.com/inveniosoftware/docker-invenio/blob/master/.travis.yml#L24

Last but not least, the docker hub builds (I skipped 3.6 for simplicity but they are the same)

Screenshot 2020-07-13 at 16 40 12


Sorry for the late reply. If possible just edit the readme to reflect this. Otherwise I can merge :)

**Comment by alexdutton** [comment]
All ready again. Have also rebased, just in case.

File: README.md
Comment (ppanero) [inline README.md:57]:
```suggestion
push to master initating a Docker Hub build that will be tagged according to the Python version.
```

**Review by ppanero — APPROVED** [review]
Small suggestion. Otherwise LGTM. I tried rebasing and applying this fix myself... but due to permissions (did you allow edit from maintainers?) I cannot push to your branch :(

**Comment by alexdutton** [comment]
@ppanero Rebased! :tada:

**Pull request merged**
